custom_components/spook/translations/pt-BR.json

```diff
@@ -0,0 +1,65 @@
+{
+  "config": {
+    "abort": {
+      "already_spooked": "Sim... bem... não. Obrigado, tchau!"
+    },
+    "step": {
+      "user": {
+        "description": "Você leu o leia-me? Você leu a licença? Você tem certeza?"
+      }
+    }
+  },
+  "issues": {
+    "user_issue": {
+      "title": "{title}",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "{title}",
+            "description": "{description}"
+          }
+        }
+      }
+    },
```

Comment:
This can be removed, there is nothing to translate in this case. So it can just fallback to the source.

```suggestion
```